.yarnrc.yml

```diff
@@ -51,32 +51,51 @@ packageExtensions:
   '@types/wordpress__plugins@2.3.7':
     peerDependencies:
       react: '*'
-  '@wordpress/annotations@1.24.8':
+  '@wordpress/annotations@2.2.2':
```

Comment:
Not related to this PR? (and all changes in this file except `@automattic/isolated-block-editor`)

Reply:
The isolated block editor update has dependencies on newer versions of wp components, so these changes are required

Reply:
So this will effectively bump lots of @wordpress dependencies, are we ok with that?

Reply:
To me, it only bumps them for this one package which is only used internally. So it feels like a safe way to split some of the work out.

that said, it doesn't _really_ matter that much if it's separate.

Reply:
You're right, this affects only the isolated block editor.

I was thinking that you'd need to dedupe lots of `@wordpress/*` dependencies, effectively affecting other places of the codebase, but looking at the changes in `yarn.lock` that's not the case. We'll have multiple versions of some `@wordpress/*` components (which may be problematic), but we already had them before this PR, so it's not a new problem.

Reply:
Plus, as more things get updated, the older versions should eventually go away